### PR TITLE
Cleanup pom.xml files - round #2

### PR DESCRIPTION
**Pull request by darxriggs**
I have found some more things to cleanup. :-)

### CODE REVIEW

**Comment by johanhaleby** [comment]
Thanks again :)

File: api/easymock/pom.xml [hunk 1]
```diff
@@ -1,14 +1,13 @@
...
     <parent>
         <artifactId>powermock-api</artifactId>
         <groupId>org.powermock</groupId>
         <version>1.6.2-SNAPSHOT</version>
     </parent>
-    <modelVersion>4.0.0</modelVersion>
     <artifactId>powermock-api-easymock</artifactId>
-    <name>powermock-api-easymock</name>
-    <version>1.6.2-SNAPSHOT</version>
+    <name>${project.artifactId}</name>
```
Comment (PascalSchumacher) [inline api/easymock/pom.xml]: Hi René,

I'm just wondering if there is any difference - other than being more explicit - between using "project.artifactId" as "name" and not adding the "name" element. Doesn't maven use the "artifactId" as name when name is not set?

Comment (darxriggs) [inline api/easymock/pom.xml]: The ``<name>`` element has no default value.

You can check with ``mvn help:effective-pom`` that if you don't specify it, it's not existing (not even inherited from the parent).

Comment (darxriggs) [inline api/easymock/pom.xml]: Ok I double checked via ``mvn site``.

If the ``<name>`` element isn't specified, the ``<artifactId>`` element's value is used.

So yes, the ``<name>`` element could be left out in this case.

Comment (PascalSchumacher) [inline api/easymock/pom.xml]: Thanks for the response. :smile: 

I thought that maven sets <name> to the <artifatId> if name is not specified, but effective-pom shows that's not the case. It appears that some plug-ins (site, assembly) use <artifactId> if <name> is unset, but maybe other plug-ins handle this differently, so your solution is the best/safest. :+1:

**Pull request merged**